components/x-content-block/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@financial-times/x-content-block",
+  "version": "0.0.0",
+  "description": "",
+  "main": "dist/ContentBlock.cjs.js",
+  "module": "dist/ContentBlock.esm.js",
+  "browser": "dist/ContentBlock.es5.js",
+  "style": "dist/ContentBlock.css",
+  "scripts": {
+    "prepare": "bower install && npm run build",
+    "build": "node rollup.js",
+    "start": "node rollup.js --watch"
+  },
+  "keywords": [
+    "x-dash"
+  ],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@financial-times/x-engine": "file:../../packages/x-engine"
+  },
+  "devDependencies": {
+    "@financial-times/x-rollup": "file:../../packages/x-rollup",
+    "bower": "^1.8.8"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Financial-Times/x-dash.git"
+  },
+  "homepage": "https://github.com/Financial-Times/x-dash/tree/master/components/x-content-block",
+  "engines": {
+    "node": ">= 6.0.0"
```

Comment:
Using `>=` could result in using a non-LTS version of Node.

I would suggest using syntax such "node": "6.x" to lock it onto a specific LTS version.

Although v12 is the current LTS version - can we specify that or is there a specific reason for using v6?

I would suggest using this as an opportunity to bump the Node version of all the consuming apps to v12 rather than keeping this new component at a low enough level to be compatible with them.

Reply:
I made it compatible with 10.x and above, because the FT app is currently pinned to 10.x.

Reply:
I think the `>=` might still allow usage of a non-LTS Node version?

Or maybe this is not an issue for components, but only the apps, i.e. does the component have to fall in line with the Node version used by the app consuming it? 🤔